actions/debootstrap_action.go

```diff
@@ -208,6 +208,11 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 	cmdline = append(cmdline, d.Mirror)
 	cmdline = append(cmdline, "/usr/share/debootstrap/scripts/unstable")
 
+	/* Make sure /etc/apt/apt.conf.d exists before running debootstrap */
+	if err := os.MkdirAll(path.Join("/etc/apt/apt.conf.d"), os.ModePerm); err != nil {
```

Comment:
If you're running outside of fakemachine this then potentially creates a directory on the host system which probably should be avoided :)